src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx

```diff
@@ -79,27 +79,29 @@ However, in this example, 5-minute wide `TIMESERIES` windows are smoothed with t
   TIMESERIES query with SLIDE BY clause
 </figcaption>
 
-## Valid NRQL syntax for `SLIDE BY` [#valid-syntax]
+## `SLIDE BY` syntax [#valid-syntax]
 
-Valid NRQL syntax for the `SLIDE BY` clause will follow the format below.
+Valid NRQL syntax for the `SLIDE BY` clause follows the format below.
 
 ```sql
 SELECT ... TIMESERIES integer1 units SLIDE BY integer2 units
 ```
 
 `integer1` specifies the sliding window width and `integer2` specifies the `SLIDE BY` interval. `units` is a time unit, such as `second`, `minute`, `hour`, or `day`. All standard NRQL time units are accepted.
 
-Here’s a real-life example. It shows 5-minute `TIMESERIES` windows with a 1-minute `SLIDE BY` interval.
+Here’s a real-life example showing 5-minute `TIMESERIES` windows with a 1-minute `SLIDE BY` interval.
 
 ```sql
 SELECT average(duration) from Transaction TIMESERIES 5 minutes SLIDE BY 1 minute
 ```
 
-## Translation from PromQL-style queries [#translate-promql]
+<Callout variant="tip">
+  When paired with `SLIDE BY`, `TIMESERIES` does not support `AUTO` or `MAX`. The `TIMESERIES` value must be an integer time unit value. In other words, `SLIDE BY AUTO` or `SLIDE BY MAX` will work, but `TIMESERIES AUTO` or `TIMESERIES MAX` followed by `SLIDE BY` and `MAX`, `AUTO`, or a specific integer time unit is not supported.
+</Callout>
 
-When applicable, a PromQL-style query is translated into a NRQL sliding window query.
+## Translation from PromQL-style queries [#translate-promql]
 
-For example, if your PromQL style query uses `rate(request_count[5m])` for the past 60 minutes with a 1-minute window overlap, the NRQL translation would be the query below.
+When applicable, a PromQL-style query is translated into a NRQL sliding window query. For example, if your PromQL style query uses `rate(request_count[5m])` for the past 60 minutes with a 1-minute window overlap, the NRQL translation would be the query below.
```

Comment:
"would be the query below" feels a bit cumbersome. How about: "...1-minute overlap. Here's that PromQL translated into NRQL:"